src/app/app.component.ts

```diff
@@ -1,10 +1,103 @@
-import { Component } from '@angular/core';
+import { Component, HostListener, OnDestroy, OnInit} from '@angular/core';
+import { Navigation, Router } from '@angular/router';
+import { Select, Store } from '@ngxs/store';
+import { Observable, Subject } from 'rxjs';
+import { takeUntil } from 'rxjs/operators';
+import { Languages } from './shared/enum/languages';
+import { Role } from './shared/enum/role';
+import { User } from './shared/models/user.model';
+import { AppState } from './shared/store/app.state';
+import { FilterState } from './shared/store/filter.state';
+import { SidenavToggle } from './shared/store/navigation.actions';
+import { NavigationState } from './shared/store/navigation.state';
+import { CheckAuth, Login, Logout } from './shared/store/registration.actions';
+import { RegistrationState } from './shared/store/registration.state';
+import { UserState } from './shared/store/user.state';
+
+
+enum RoleLinks {
+  provider = 'організацію',
+  parent = 'дитину'
+}
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
-export class AppComponent {
+
+
+export class AppComponent implements OnInit, OnDestroy{
+
+  readonly Languages: typeof Languages = Languages;
+  selectedLanguage: string = 'uk'
```

Comment:
selected lang should be get from storage

Reply:
please let me know when you could clarify about this question